### DISCUSSION TITLE
How to make git blame date format say "14 months ago" instead of "1 year ago"

### DISCUSSION THREAD

**Discussion by Cheldon** [gitkraken/vscode-gitlens#1539] [Q&A] [answered]
Is there a way to adjust when the git blame date format switches from months to years?
It is really useful to see the "ago" time in days, weeks, then months, but once it goes to years it loses a lot of precision. "1 year ago" can mean 365 days or it can mean (2 years - 1 day).
Is there a way to use `agoOrDate` or `gitlens.defaultDateSource` so certain month ranges are not truncated to years?
I think knowing something was 14 months ago would be more useful than an ambiguous  "1 year" = 12-24 months ago. But after 24 months I wouldn't want as much precision. 

Is there an option to show "1 year, 2 months ago"? That would be a great (probably better) option as well.

**Comment by eamodio** [comment, marked as answer]
There isn't an option currently, sorry. Would you mind opening a feature request for that -- I think it's a good idea.

**Reply by Cheldon** [reply]
https://github.com/eamodio/vscode-gitlens/issues/1546
